src/gui/gui.h

```diff
@@ -162,6 +162,7 @@ class GUI final {
 
     float m_renderRatio = 3.0f / 4.0f;
     bool &m_fullscreen = {settings.get<Fullscreen>().value};
+    std::string isoFilename;
```

Comment:
If you're adding new member variables, please use the m_ prefix :-)

Also, as described in the general comment, this may not be the right location for this.

And maybe we just want to use an std::filename::path for it?